cfgov/unprocessed/apps/youth-employment-success/js/reducers/choice-reducer.js

```diff
@@ -0,0 +1,26 @@
+import { actionCreator } from '../util';
+
+const initialState = '';
+const TYPES = {
+  UPDATE_ROUTE_OPTION_CHOICE: 'UPDATE_ROUTE_OPTION_CHOICE'
+};
+const updateRouteChoiceAction = actionCreator(
+  TYPES.UPDATE_ROUTE_OPTION_CHOICE
+);
+
+function choiceReducer( state = initialState, action ) {
+  switch ( action.type ) {
```

Comment:
Is having a switch here just convention, since an if would work here also?

Reply:
Yeah the `switch` is a convention, in case more actions are added, although it is unlikely to happen in this case.